app/uploaders/image_uploader.rb

```diff
@@ -13,6 +13,10 @@ def store_dir
     "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
   end
 
+  def url
+    "http://localhost:3000/#{super}"
```

Comment:
viết như này khi thay đổi env thì đổi tay à (dull) move to Settings....

Reply:
ok anh